.gitlab-ci.yml

```diff
@@ -28,22 +28,19 @@ Cluster Connect:
     paths:
       - .kube/
 
-Login_Workflow_Test:
+Cypress-e2e:
   when: always
   image: cypress/base:10
   stage: cypress-test
   tags:
     - litmus-portal
   script:
-    - cd CypressE2E
-    - npm ci
-    - cd ../
-    - git clone -b litmus-portal https://github.com/litmuschaos/litmus.git
-    - cd litmus/litmus-portal/frontend
-    - npm ci
-    - npm start &
+    - cd CypressE2E && npm ci
+    - cd ../ && git clone -b litmus-portal https://github.com/litmuschaos/litmus.git
+    - cd litmus/litmus-portal/frontend && npm i
```

Comment:
`&& npm i` or `&& npm ci` ?

Reply:
Both will work,not a problem.
npm i is fast that's why used for frontend dependencies.